client/components/mma/accountoverview/AccountOverview.tsx

```diff
@@ -107,33 +108,33 @@ const AccountOverviewPage = ({ isFromApp }: IsFromAppProps) => {
 			b.subscription.start.localeCompare(a.subscription.start),
 	);
 
-	const productCategories = [
+	const allProductCategories = [
 		...allActiveProductDetails,
 		...allCancelledProductDetails,
-	]
-		.map(
-			(product: ProductDetail | CancelledProductDetail) =>
-				product.mmaCategory,
-		)
-		.filter((value, index, self) => self.indexOf(value) === index);
-
+	].map((product: ProductDetail | CancelledProductDetail) => {
+		if (product.mmaCategory === 'recurringSupport') {
+			return 'subscriptions';
+		}
+		return product.mmaCategory;
+	});
+	const uniqueProductCategories = [...new Set(allProductCategories)];
```

Comment:
didn't realise this was at the same level - add gaps?
```suggestion

	const uniqueProductCategories = [...new Set(allProductCategories)];
	
```